skins/cat17/src/sass/components/fonts/icomoonwikimedia/style.scss

```diff
@@ -20,170 +20,172 @@
   font-variant: normal;
   text-transform: none;
   line-height: 1;
-  // custom base size
-  font-size: 24px;
   /* Better Font Rendering =========== */
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-.icon-payment-sofort {
-  &:before {
-    content: $icon-payment-sofort;
-  }
+@function unicode($str) {
+  @return unquote("\"") + $str + unquote("\"");
```

Comment:
I don't understand why this change is necessary, could you put in a comment here?

Reply:
The comment in main.scss  mentions that style.scss is generated by icomoon. I'm not sure what comment exactly to add to this line here.